### ISSUE TITLE
Question about training from scratch.

### ISSUE THREAD

**Issue by z1358** [cyvius96/few-shot-meta-baseline#26]
What a great work! 
I have some questions about the training from scratch in table 5. 
Do you mean training the second stage(encoder + cosine similarity) without the first stage? Just like change the `load_encoder` and `load` to `None` in the `train_meta_mini.yaml` ? If so, what is the details about optimizer? Is it the same as the first stage?

**Comment by cyvius96** [comment]
Thanks for your interest in our work!

I currently do not have access to the original config for this training-from-scratch experiments. Since its performance is limited, we did not release its config.
I remember the config should be like, in train_meta_mini.yaml:
* delete the row "load_encoder"
* set "ep_per_batch" to 8 (or 16)
* set "train_batches" to 300
* set "max_epoch" to 180, set "lr: 0.1" and add "milestones: [90] (or [90, 150])" in "optimizer_args"

The parameters above may not be accurate. The general idea is to train the model with initial lr 0.1 and decay it when validation loss gets plateau. You may adjust these parameters according to the training plot.

**Comment by z1358** [comment]
OK, I will try your suggestion. Thanks for your reply!
